core/src/main/java/org/infinispan/configuration/cache/Configurations.java

```diff
@@ -19,11 +19,19 @@ private Configurations() {
    }
 
    public static boolean isOnePhaseCommit(Configuration cfg) {
+      // Otherwise pessimistic transactions will be one phase commit
+      if (cfg.memory().size() > 0 && cfg.memory().evictionType().isExceptionBased()) {
```

Comment:
Perhaps you could extract this statement into a method. This way, the method name would explain everything and the comment above would be redundant.

Reply:
What would you recommend the method name to be then?